### PR TITLE
fix(calendar): enhance accessibility telerik/kendo#6783

### PR DESCRIPTION
**Pull request by dbalinov**
Related to telerik/kendo#6783

### CODE REVIEW

File: src/kendo.calendar.js [hunk 1]
```diff
@@ -950,7 +951,8 @@ var __meta__ = { // jshint ignore:line
                 today = new DATE(),
                 html = '<table tabindex="0" role="grid" class="k-content" cellspacing="0" data-start="' + toDateString(start) + '"><thead><tr role="row">';
                 if (isWeekColumnVisible) {
-                    html += '<th scope="col" class="k-alt"></th>';
+                    // an empty <th> is not 508 compliant and it should have some text content
+                    html += '<th scope="col" class="k-alt"><span style="visibility: hidden;">.</span></th>';
```
Comment (dbalinov) [inline src/kendo.calendar.js]: If I add visibility: hidden directly to th tag the gray background color of th is also hidden. This is the reason for the additional span element.

Comment (Stamo-Gochev) [inline src/kendo.calendar.js]: Can @theOrlin give some other idea about that?

Comment (Xizario) [inline src/kendo.calendar.js]: Just:
`<th scope="col" class="k-alt">&nbsp;</th>`
More correct fix would be to use something like calendar.standard.`weekAbbreviation` that to be [localized](https://github.com/telerik/kendo/blob/master/src/cultures/kendo.culture.en-US.js#L30).

Comment (dbalinov) [inline src/kendo.calendar.js]: @Xizario:
`<th scope="col">&nbsp;</th>` this is not working.
We had the same scenario for the grid and @Stamo-Gochev had some concerns for longer abbreviation than "."

**Comment by dbalinov** [comment]
The new approach we decided to use for achieving accessibility for empty `th` tag when weeks are enabled is the following:
* Add message variable for `th`
* Set default value to empty string so we are backward compatible

If you want to set the value, the following messages block can be added to the `options `object:
```json
{
    "weekNumber": true,
    "messages": {
         "weekColumnHeader": "W"
     }
}
```

**Comment by Xizario** [comment]
The messages should be added into the API as well.

**Pull request merged**